data/virtualization/autoyast/guest_12.xml.ep

```diff
@@ -51,7 +51,7 @@
     <arch>{{ARCH}}</arch>
     <release_type/>
     </addon>
-    % if ($ltss_code) {
+    % if ($ltss_code && $vm_name !~ /ltsses/) {
```

Comment:
A minor suggestion, it maybe more name telling with `ltss-es`? Given that not all know the details of 12sp5 ltss es,  suggest to add a comment here to explain a little bit and add the progress ticket.

Reply:
Yes, ltss-es is better name.